src/node/main.ts

```diff
@@ -129,6 +130,15 @@ export const runCodeServer = async (args: DefaultedArgs): Promise<http.Server> =
     logger.info("  - Connected to cloud agent")
   }
 
+  try {
+    const port = parseInt(serverAddress.split(":").pop() as string, 10)
```

Comment:
This code won't work correctly with IPv6, I'm not sure if code-server currently supports IPv6 though. IMO it's always better to use the URL object (or similar) to parse hostname/port

Reply:
Good point... will do!

Reply:
I forgot we don't support IPv6 for our webviews anyways, thanks @jsjoeio for the reminder. So I think this is good to go as-is